src/stories/button-favourite/ButtonFavourite.tsx

```diff
@@ -6,7 +6,7 @@ interface ButtonFavouriteProps {
 
 export const ButtonFavourite = ({ fill }: ButtonFavouriteProps) => {
   return (
-    <button className="button-favourite">
+    <button type="button" aria-label="Add to favourites" className="button-favourite">
```

Comment:
Consider if the text for `aria-label` should be in Danish or English. I am not sure if we have a established practice here.